Gemfile

```diff
@@ -41,7 +41,7 @@ gem 'active-fedora'
 gem 'blacklight', '~> 5.9.0' # TODO: BL >= 5.10.x has new deprecation warnings vs <= 5.9.x, will investigate and unpin after current upgrade stuff has settled
 gem 'blacklight-hierarchy'
 gem 'blacklight-marc'
-gem 'dor-services', '~> 5.0', :git => 'https://github.com/sul-dlss/dor-services.git', :branch => 'develop'
+gem 'dor-services', '~> 5.0', :git => 'https://github.com/sul-dlss/dor-services.git', :branch => 'indexing-service'
```

Comment:
once https://github.com/sul-dlss/dor-services/pull/153 is merged, this can go back to develop.  probably still just a bit too early to start pointing to released versions.